ClientApp/viking-vault/src/components/styles.css

```diff
@@ -7,7 +7,7 @@
     transition: 0.5s;
     position: absolute;
     box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
-    width: 150px;
+    width: 60px;
 }
 
 .sidebar-false{
```

Comment:
Please avoid using this class naming of "base-" + "true"/"false", it's really confusing, you could for example use `.sidebar.expanded` or `sidebar-collapsed` and `sidebar-expanded` (if that's what this true/false means when referring to the sidebar)